swagger.yaml

```diff
@@ -3965,18 +3822,18 @@ definitions:
               type: string
               format: uuid
             role:
-              $ref: '#/definitions/host-role-update-params'
-      hosts_names:
+              $ref: '#/definitions/agent-role-update-params'
+      agents_names:
         type: array
-        description: The desired hostname for hosts associated with the cluster.
+        description: The desired agentname for agents associated with the cluster.
         x-nullable: true
         items:
           type: object
           properties:
             id:
               type: string
               format: uuid
-            hostname:
+            agentname:
```

Comment:
isn't renaming hostname to agentname a little bit confusing?this field defines the hostname of the machine, event if the machine is called an agent